packages/cloud-cognitive/src/components/AddSelect/_add-select.scss

```diff
@@ -440,7 +448,7 @@
 
   .#{$block-class}__multi .#{$pkg-prefix}--empty-state {
     max-width: 16rem;
-    margin-top: $layout-04;
+    margin-top: $spacing-04;
```

Comment:
`$layout-04` was 3rem so the equivalent spacing should actually be `$spacing-09`.